ci/scripts/backwards-compat-test.sh

```diff
@@ -76,13 +74,20 @@ setup_old_cluster() {
   git config --global --add safe.directory /risingwave
   git checkout "v${OLD_VERSION}"
   cargo build -p risedev
+  echo "--- Get RisingWave binary for $OLD_VERSION"
   OLD_URL=https://github.com/risingwavelabs/risingwave/releases/download/v${OLD_VERSION}/risingwave-v${OLD_VERSION}-x86_64-unknown-linux.tar.gz
+  set +e
   wget $OLD_URL
-  tar -xvf risingwave-v${OLD_VERSION}-x86_64-unknown-linux.tar.gz
-  mv risingwave target/debug/risingwave
-
-  echo "--- Start cluster on tag $OLD_VERSION"
-  git config --global --add safe.directory /risingwave
+  set -e
+  if [[ "$?" -ne 0 ]]; then
+    echo "Failed to download ${OLD_VERSION} from github releases, build from source later"
```

Comment:
Later on in the script we will call `risedev d`, which should build the rw binaries, since they are missing.